mod/ignored.php

```diff
@@ -1,52 +0,0 @@
-<?php
-
-use Friendica\App;
-use Friendica\Database\DBA;
-use Friendica\Model\Item;
-
-function ignored_init(App $a)
-{
-	if (!local_user()) {
-		exit();
-	}
-
-	if ($a->argc > 1) {
-		$message_id = intval($a->argv[1]);
-	}
-
-	if (empty($message_id)) {
-		exit();
-	}
-
-	$thread = Item::selectFirstThreadForUser(local_user(), ['uid', 'ignored'], ['iid' => $message_id]);
-	if (!DBA::isResult($thread)) {
-		exit();
-	}
-
-	// Numeric values are needed for the json output further below
-	$ignored = ($thread['ignored'] ? 0 : 1);
-
-	if ($thread['uid'] != 0) {
```

Comment:
I transformed this one to the new method, but the field isn't used in this "if" statement anymore ..